ttnn/cpp/ttnn/operations/data_movement/concat/device/concat_device_operation.cpp

```diff
@@ -123,7 +123,9 @@ Tensor concat_impl(std::vector<Tensor> &input_tensors, const std::int64_t dim, c
                             "Current concat implementation requires aligned last dim when concatting on last dim");
                     }
                 }
-                Layout target_layout = Layout::TILE;
+                // row major should default to row major and tilized to tilized implementations, but the below loop turned RM to tilized when possible
+                Layout target_layout = input_tensors[0].get_layout();
+                // this should be dead code when instantiating layout to match the input
```

Comment:
Currently the instantiated layout for an output tensor does match the layout of the input tensor (see `create_output_tensors`). So is this a dead code already, or am I missing something?